pkg/virt-controller/services/template.go

```diff
@@ -600,10 +610,29 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 			},
 		})
 
+		reqMemDiff := resource.NewScaledQuantity(0, resource.Kilo)
```

Comment:
What does this cover. What does this diff between?

Reply:
Well, that's just a variable initialization.
you should look at lines 621-624 to see that the difference is between the guest memory and the requested memory for the compute container

Reply:
Later in the code, the value of the variable does not match its name. Could you initialize a new variable which name would reflect that it has the total memory, not only the diff?